pkg/validation/spec/gnostic_test.go

```diff
@@ -294,158 +292,6 @@ func BenchmarkGnosticConversion(b *testing.B) {
-		},
-		{
-			Name:    "OAuth2 Access Code",
-			Pattern: `{"type": "oauth2", "flow": "accessCode", "authorizationUrl": "https://google.com", "tokenUrl": "https://google.com", "scopes": {"scope1": "a scope", "scope2": "a scope"}, "description": "cool oauth2 auth"}`,
-		},
-	}
-
-	for _, p := range patterns {
-		t.Run(p.Name, func(t *testing.T) {
-			// Parse JSON into yaml
-			var nodes yaml.Node
-			if err := yaml.Unmarshal([]byte(p.Pattern), &nodes); err != nil {
-				t.Error(err)
-				return
-			} else if len(nodes.Content) != 1 {
-				t.Errorf("unexpected yaml parse result")
-				return
-			}
-
-			root := nodes.Content[0]
-
-			parsed, err := openapi_v2.NewSecurityDefinitionsItem(root, compiler.NewContextWithExtensions("$root", root, nil, nil))
-			if err != nil {
-				t.Error(err)
-				return
-			}
-
-			converted := SecurityScheme{}
-			if err := converted.FromGnostic(parsed); err != nil {
-				t.Error(err)
-				return
-			}
-
-			// Ensure that the same JSON parsed via kube-openapi gives the same
-			// result
-			var expected SecurityScheme
-			if err := json.Unmarshal([]byte(p.Pattern), &expected); err != nil {
-				t.Error(err)
-				return
-			} else if !reflect.DeepEqual(expected, converted) {
-				t.Errorf("expected equal values: %v", cmp.Diff(expected, converted, SwaggerDiffOptions...))
-				return
-			}
-		})
-	}
-}
-
-// Ensure all variants of Parameter are being exercised by tests
-func TestParamVariants(t *testing.T) {
-	type TestPattern struct {
-		Name    string
-		Pattern string
-	}
-
-	patterns := []TestPattern{
-		{
-			Name:    "Body Parameter",
-			Pattern: `{"in": "body", "name": "myBodyParam", "schema": {}}`,
-		},
-		{
-			Name:    "NonBody Header Parameter",
-			Pattern: `{"in": "header", "name": "myHeaderParam", "description": "a cool parameter", "type": "string", "collectionFormat": "pipes"}`,
-		},
-		{
-			Name:    "NonBody FormData Parameter",
-			Pattern: `{"in": "formData", "name": "myFormDataParam", "description": "a cool parameter", "type": "string", "collectionFormat": "pipes"}`,
-		},
-		{
-			Name:    "NonBody Query Parameter",
-			Pattern: `{"in": "query", "name": "myQueryParam", "description": "a cool parameter", "type": "string", "collectionFormat": "pipes"}`,
-		},
-		{
-			Name:    "NonBody Path Parameter",
-			Pattern: `{"required": true, "in": "path", "name": "myPathParam", "description": "a cool parameter", "type": "string", "collectionFormat": "pipes"}`,
-		},
-	}
-
-	for _, p := range patterns {
-		t.Run(p.Name, func(t *testing.T) {
-			// Parse JSON into yaml
-			var nodes yaml.Node
-			if err := yaml.Unmarshal([]byte(p.Pattern), &nodes); err != nil {
-				t.Error(err)
-				return
-			} else if len(nodes.Content) != 1 {
-				t.Errorf("unexpected yaml parse result")
-				return
-			}
-
-			root := nodes.Content[0]
-
-			ctx := compiler.NewContextWithExtensions("$root", root, nil, nil)
-			parsed, err := openapi_v2.NewParameter(root, ctx)
```

Comment:
`NewContextWithExtensions` and `NewParameter`'s  uses `yaml.Node` from `gopkg.in/yaml.v3` in their method signature so we cannot use this method.